Misc/NEWS.d/next/Library/2018-08-04-12-26-11.bpo-28009.4JcHZb.rst

```diff
@@ -0,0 +1 @@
+Fix uuid.getnode() on AIX - to use proper MAC_DELIM character
```

Comment:
```suggestion
Fix uuid.getnode() on AIX to use proper MAC_DELIM character
and to handle the potential lack of leading zeroes in MAC address
segments.
```

Reply:
I forget what was supposed to be there.  This works too.